File: server/hotrod/src/test/scala/org/infinispan/server/hotrod/event/HotRodDistributedEventsTest.scala
package org.infinispan.server.hotrod.event

import org.testng.annotations.Test
import org.infinispan.configuration.cache.CacheMode

/**
 * @author Galder Zamarreño
 */
@Test(groups = Array("functional"), testName = "server.hotrod.event.HotRodDistributedEventsTest")
class HotRodDistributedEventsTest extends AbstractHotRodClusterEventsTest {
   cacheMode = CacheMode.DIST_SYNC
}

File: server/hotrod/src/test/java/org/infinispan/server/hotrod/event/HotRodDistributedEventsTest.java
package org.infinispan.server.hotrod.event;

import org.infinispan.configuration.cache.CacheMode;
import org.testng.annotations.Test;

/**
 * @author Galder Zamarreño
 */
@Test(groups = "functional", testName = "server.hotrod.event.HotRodDistributedEventsTest")
public class HotRodDistributedEventsTest extends AbstractHotRodClusterEventsTest {
   public HotRodDistributedEventsTest() {
      cacheMode = CacheMode.DIST_SYNC;
   }
}
